### PR TITLE
Improve self.id page

### PR DESCRIPTION
**Pull request by michaelsena**

### CODE REVIEW

File: docs/tools/identity/self-id.md [hunk 1]
```diff
@@ -1,30 +1,35 @@
...
 
-IDX:
+**Basic profiles**: Self.ID allows users to create and/or edit a basic profile for their 3ID DID. To achieve this, Self.ID integrates with the [IDX SDK](./idx.md) to create and query [IDX records](./idx.md#records) that conform to the [Basic Profile definition and schema (CIP-19)](https://github.com/ceramicnetwork/CIP/blob/main/CIPs/CIP-19/CIP-19.md).
 
-IdentityLink:
+**Link Web3 accounts**: Self.ID allows users to link many different blockchain accounts to their 3ID. To achieve this, Self.ID integrates with the [IDX SDK](./idx.md) to create and query [IDX records](./idx.md#records) that conform to the [Crypto Accounts definition and schema (CIP-21)](https://github.com/ceramicnetwork/CIP/blob/main/CIPs/CIP-21/CIP-21.md). Each user's Crypto Accounts record in IDX contains a list of [streamIDs](../../learn/glossary.md#streamid) for the [Caip10Links](../../streamtypes/caip-10-link/overview.md) of their 3ID.
```
Comment (PaulLeCam) [inline docs/tools/identity/self-id.md:20]: We don't have this directly in self.id at the moment so maybe we can leave this section out until it's implemented?

Comment (michaelsena) [reply docs/tools/identity/self-id.md:20]: We kind of provide this through 3ID Connect. Maybe i could make that more clear. Think that would address the concern?

Comment (PaulLeCam) [reply docs/tools/identity/self-id.md:20]: I don't think the flow is obvious now, especially coming from self.id that might require "logging out" and connecting again via 3ID Connect, so it's not really user-friendly at this point.

Comment (michaelsena) [reply docs/tools/identity/self-id.md]: Okay removed for now

**Comment by michaelsena** [comment]
Will merge once #92 is merged, since this page uses relative links to pages created in that PR.

**Pull request merged**